R/quotation.R

```diff
@@ -413,6 +413,12 @@ endots <- function(call,
 #'   argument and `width` as the second argument.
 #' @export
 exprs_auto_name <- function(exprs, width = NULL, printer = expr_name) {
+  if (is_installed("dplyr") &&
+        utils::packageVersion("dplyr") < "0.7.99" &&
```

Comment:
The `is_reference()` call is two orders of magnitude faster than `utils::packageVersion()` .

Perhaps we could just check if `printer` evaluates to `dplyr:::tidy_text` (if that exists), and make sure we rename in dev dplyr?

Reply:
we could do that,`tidy_text()` should be removed from dplyr